Getting Hired: Social Networking Lesson (#26307)
TheOdinProject/curriculum: getting_hired/preparing_for_job_search/professional_networking.md

### What is a professional network?

Your professional network is not the people you follow on social media, nor is it necessarily your friends. It is the people with whom you have made a genuine connection, whether in-person or online, because you have a mutual interest and can gain more value through being connected than you would otherwise have alone. These are the people you can ask for help, and in turn help out.

The network that will help you on your journey to becoming a software developer will mostly consist of people working in software development and adjacent roles, like recruitment or management. Take stock of your network in this field now. Chances are you might benefit from expanding it?

### Why is a professional network useful?

You can probably think of instances where you were stuck on a task and asked someone for help. This was you exercising your network. You can do the same with your software developer network to help you with many things like gather information, make decisions, and find a job.

> When it comes to information gathering, search engines and AI can be very helpful. But for anything with a bit of nuance, it’s often easier to ask a real human. For example, you might want to [conduct an informational interview](https://blog.get-merit.com/break-into-tech-with-informational-interviewing/) with a senior developer to learn about what they do and how they broke into the industry. You might be seeking feedback on your portfolio. Or, looking to get some career progression advice. Interacting with someone who has experience in what you seek can be far more beneficial because you can form a dialogue, ask follow-up questions, and get real-time feedback. 

Reviewer: Just looked at the resource. I think that captures it well!

I'm thinking it could go in the Identify Your Options section as an option for folks to consider.

Author: Yes that makes sense. Done.

* [Discord](https://discord.com/) - there’s likely a discord channel dedicated to a given language or framework where you can ask questions and get help. You can sign up for Discord and search for what you are interested in e.g., NodeJS, Ruby on Rails, and request to join.
* Open Source projects - when you have the skillset and confidence to contribute to open-source projects, you will be working directly with project maintainers and contributors. It is a great way to connect with other developers while working towards a common good.

### Build Genuine Connections

The economy of professional networking rewards authenticity and mutual respect. If you are going to ask a favor of someone, you are far more likely to get help if you have developed a genuine connection. That is to say, please do not approach people ‘cold’ and ask them out of the blue to connect with you, or for a favor like ‘help me get this job in your company’ or ‘please be my mentor’. This may be perceived as disrespectful. People are busy, and they reserve their limited time for helping those they know and respect.

> Your best strategy is to start conversations with a genuine interest in the person and connect on a mutual interest. For example, you could review someone’s profile on LinkedIn and find something in common, or begin a conversation in-person with questions related to what the person does, how they got to where they are, and what challenges they face in their daily work. Make it about them, not you. Show that you are interested in them as a person, not that you want them to give you something. Actively listen to what they say and ask follow-up questions and even offer to help with something if you are in a position to do so. Think of the interaction like dating, you don’t ask someone to be your partner in the first question (do you? lol), so the same applies here. If your initial conversation goes well, then you can follow up with a request to connect.

Reviewer: I feel like this paragraph is hinting at informational interviews. Not suggesting we write an explainer on that from scratch (unless you want to) but linking to something that explains informational interviews would feel good here.

Author: Yeah good call. That adds value here.

### Introduction

A robust professional network is an irreplaceable asset on your journey to becoming a web developer, and beyond. As social beings we are wired to connect with others, receive help, and give help. This is no different in the software ecosystem where countless communities take part in the mutually beneficial transactions of networking.

> One significant benefit of your network is that it might help land you your first job in software. This is not a guarantee, but it is the lived experience of many developers. As with all things, you get back what you put in, so the more developed your network is, the better your odds it will provide for you. 

Reviewer 2: might help _**you**_ land ~~you~~ your first

Author: good catch

> One significant benefit of your network is that it might help land you your first job in software. This is not a guarantee, but it is the lived experience of many developers. As with all things, you get back what you put in, so the more developed your network is, the better your odds it will provide for you. 

Reviewer 2: it will provide **_benefit_** for you.

In this lesson we cover what a professional network is, how it might benefit you, and some strategies to consider when to building one. Making genuine connections with people takes time, however, so we recommend you start investing time and effort sooner than later.

### What is a professional network?

> Your professional network is not the people you follow on social media, nor is it necessarily your friends. It is the people with whom you have made a genuine connection, whether in-person or online, because you have a mutual interest and can gain more value through being connected than you would otherwise have alone. These are the people you can ask for help, and in turn help out.

Reviewer 2: and in turn help **_you_** out.

Author: Ah I meant they can help you, and you can help them. I reworded slightly.

Think of it from the company’s perspective. Hiring is a pain. In a perfect world, a great candidate would show up instantly any time a new job opened up. This would save them going through the process of posting the job, screening applicants, marking technical assessments and conducting interviews. This all takes time and is very expensive. So it stands to reason why most companies would prefer to hire through the hidden jobs market. 

So, how do you tap into the hidden jobs market? Through your network. If your network sees you as a trusted individual with a particular skill set, you can be sure they will think of you when matching opportunities arise. You might ask around about potential jobs and receive a heads up of a new opportunity at someone’s company. Better still, you could be recommended for a particular role. Many companies give bonuses to staff who refer a candidate that gets hired. So, the incentive to network and pass names along goes both ways.

> This is not all to say you should refrain entirely from applying on posted jobs. These can still be worthwhile to pursue, but we recommend quality over quantity and seeking companies that you find interesting and have core values you identify with. You will be better off sending fewer, tailored applications over hundreds of the same application in a shotgun-like approach. We cover more on this in the Getting Hired course.

Reviewer 2: from applying ~~on~~ _**to**_ posted jobs

#### Start Now
Your network will take time to build. The sooner you start to make connections, the better. Starting now means you will have more of a network to leverage when you need to rely on it. Consider proactively allocating a percentage of your web development study time to this activity. This can act as a nice breather between the mental workout of coding, and you may even combine networking and learning during activities like attending local meetups and conferences.

#### Identify Your Options
Your time is limited and your options are endless. So, you should understand what options you have at your disposal and select one or two to start with. 

In-person networking opportunities will vary based on your location. Larger towns and cities will likely have well-established tech-related events and groups, whereas smaller communities may be more limited. To get a better sense of what’s going on around you, consider the following:

> * Check [MeetUp](https://www.meetup.com/) and/or Google for communities focused on a given technology.

Reviewer 2: Suggest adding LinkedIn as a good option to find local events/groups.

Author: done

* [X](https://twitter.com/) - this is a great platform to keep a finger on the pulse of a given domain. You can follow companies and individuals, engage with their posts, and create your own content.
* [Discord](https://discord.com/) - there’s likely a discord channel dedicated to a given language or framework where you can ask questions and get help. You can sign up for Discord and search for what you are interested in e.g., NodeJS, Ruby on Rails, and request to join.
* Open Source projects - when you have the skillset and confidence to contribute to open-source projects, you will be working directly with project maintainers and contributors. It is a great way to connect with other developers while working towards a common good.

> Once you have built up your network a little, a great way to find out good information quickly is through [informational interviews](https://blog.get-merit.com/break-into-tech-with-informational-interviewing/). An informational interview is the formal name for a chat with someone who is in a position you want to be in, or are interested in learning about. You get to ask them questions, such as "What path did you take to get to this point?", "What do you do in a typical work day?", "What advice would you give to someone like me?", and "Tell me about the challenges you face in your role". The questions you ask should help you find out if a particular role or company is the right fit for you and if you need to make any adjustments in your strategy.

Reviewer: One tiny addition that I think would be useful here is to express how we can leverage contacts we make in informational interviews into professional contacts.

Author: Ah right. Yes I was thinking we would recommend making a contact first then interviewing, but consitent with the resource provided, that would make sense.

Author: Done!

One significant benefit of your network is that it might help you land your first job in software. This is not a guarantee, but it is the lived experience of many developers. As with all things, you get back what you put in, so the more developed your network is, the better your odds it will provide a benefit for you. Making genuine connections with people takes time, however, so we recommend you start investing time and effort sooner than later.

### Lesson overview

This lesson covers the following topics:

*   What a professional network is
*   How a professional network might benefit you
*   Strategies and actions to consider when building and maintaining your professional network

### What is a professional network?

Your professional network is not the people you follow on social media, nor is it necessarily your friends. It is the people with whom you have made a genuine connection, whether in-person or online, because you have a mutual interest and can gain more value through being connected than you would otherwise have alone. These are the people you can ask for help, and in turn you can help out later.

> The network that will help you on your journey to becoming a software developer will mostly consist of people working in software development and adjacent roles, like recruitment or management. Take stock of your network in this field now. Chances are you might benefit from expanding it?

Reviewer 3: ```suggestion
 The network that will help you on your journey to becoming a software developer will mostly consist of people working in software development and adjacent roles, like recruitment or management. Take stock of your network in this field now. Chances are, you might benefit from expanding it.
```

### Why is a professional network useful?

> You can probably think of instances where you were stuck on a task and asked someone for help. This was you exercising your network. You can do the same with your software developer network to help you with many things like gather information, make decisions, and find a job.

Reviewer 3: ```suggestion
You can probably think of instances where you were stuck on a task and asked someone for help. This is an example of exercising your network. You can do the same with your software developer network to help you gather information, make decisions, and find a job.
```

> When it comes to information gathering, search engines and AI can be very helpful. But for anything with a bit of nuance, it’s often easier to ask a real human. For example, you might want to conduct an informational interview with a senior developer to learn about what they do and how they broke into the industry. You might be seeking feedback on your portfolio. Or, looking to get some career progression advice. Interacting with someone who has experience in what you seek can be far more beneficial because you can form a dialogue, ask follow-up questions, and get real-time feedback. 

Reviewer 3: ```suggestion
When it comes to information gathering, search engines and AI can be very helpful. But when more nuance is needed, it can be easier to ask a real human. For example:
-  You might want to conduct an informational interview with a senior developer to learn about what they do and how they broke into the industry.
-  You might be seeking feedback on your portfolio.
-  You may be looking to get some career progression advice.
Interacting with someone who has experience in what you seek can be beneficial because you can form a dialogue, ask follow-up questions, and get real-time feedback. 
```

> Your network can also help you make decisions. Have you ever had decision paralysis? This is when you are presented with so many options, you cannot choose. This happens a lot in tech, especially for newcomers, given the sheer number of languages, frameworks and tools to choose from. Sometimes it’s easier to canvas the people you trust and take their recommendations. By doing this we are gathering social proof. If someone we know recommends something, we are more likely to choose it (it’s also why you read reviews before buying stuff online!).

Reviewer 3: consider "social proof" - this is somewhat of a technical term.

> When it comes to finding a job, your network can be the most valuable asset you have. Depending on the source, up to [80% of all jobs are not posted online](https://www.forbes.com/sites/dianatsai/2017/10/02/80-of-jobs-are-not-on-job-boards-heres-how-to-find-them/?sh=29ada3fcd455). This means you are only seeing about 20% of all open jobs when you are browsing job boards, and the rest are in the ‘hidden jobs market’. Now, these percentages may vary, but it’s a safe bet that the hidden jobs market is real, and likely significant. If your strategy does not attempt to tap into the hidden jobs market, you are undermining your chances of success and will be stuck competing with many applicants for fewer jobs.

Reviewer 3: ```suggestion
When it comes to finding a job, your network can be the most valuable asset you have. Depending on the source, up to [80% of all jobs are not posted online](https://www.forbes.com/sites/dianatsai/2017/10/02/80-of-jobs-are-not-on-job-boards-heres-how-to-find-them/?sh=29ada3fcd455). This means you are only seeing about 20% of all open jobs when you are browsing job boards, and the rest are in the ‘hidden jobs market’. Now, these percentages may vary, but it’s a safe bet that the hidden jobs market is real, and likely significant. If your strategy does not attempt to tap into this market, you are undermining your chances of success and will be stuck competing with many applicants for fewer jobs.
```

Think of it from the company’s perspective. Hiring is a pain. In a perfect world, a great candidate would show up instantly any time a new job opened up. This would save them going through the process of posting the job, screening applicants, marking technical assessments and conducting interviews. This all takes time and is very expensive. So it stands to reason why most companies would prefer to hire through the hidden jobs market. 

> So, how do you tap into the hidden jobs market? Through your network. If your network sees you as a trusted individual with a particular skill set, you can be sure they will think of you when matching opportunities arise. You might ask around about potential jobs and receive a heads up of a new opportunity at someone’s company. Better still, you could be recommended for a particular role. Many companies give bonuses to staff who refer a candidate that gets hired. So, the incentive to network and pass names along goes both ways.

Reviewer 3: ```suggestion
So, how do you tap into the hidden jobs market? Through your network. If your network sees you as a trusted individual with a particular skill set, you can be sure they will think of you when matching opportunities arise. You might ask around about potential jobs and receive a heads up of a new opportunity at someone’s company. Better still, you could be recommended for a particular role. Many companies even give bonuses to staff who refer a candidate that gets hired, so the incentive to network and pass names along goes both ways.
```

> This is not all to say you should refrain entirely from applying to posted jobs. These can still be worthwhile to pursue, but we recommend quality over quantity and seeking companies that you find interesting and have core values you identify with. You will be better off sending fewer, tailored applications over hundreds of the same application in a shotgun-like approach. We cover more on this in the Getting Hired course.

Reviewer 3: ```suggestion
This is not to say you should refrain entirely from applying to posted jobs. These are definitely still worthwhile to pursue, but it's still better to choose quality over quantity by seeking companies that you find interesting and have core values you identify with. You will be better off sending fewer, tailored applications over hundreds of the same application in a shotgun-like approach. We cover more on this in the Getting Hired course.
```

#### Start Now
Your network will take time to build. The sooner you start to make connections, the better. Starting now means you will have more of a network to leverage when you need to rely on it. Consider proactively allocating a percentage of your web development study time to this activity. This can act as a nice breather between the mental workout of coding, and you may even combine networking and learning during activities like attending local meetups and conferences.

#### Identify Your Options
Your time is limited and your options are endless. So, you should understand what options you have at your disposal and select one or two to start with. 

In-person networking opportunities will vary based on your location. Larger towns and cities will likely have well-established tech-related events and groups, whereas smaller communities may be more limited. To get a better sense of what’s going on around you, consider the following:

> * Check [MeetUp](https://www.meetup.com/), [LinkedIn](https://www.linkedin.com/), and/or Google for local communities focused on a given technology.

Author: Done!

### Build Genuine Connections

The economy of professional networking rewards authenticity and mutual respect. If you are going to ask a favor of someone, you are far more likely to get help if you have developed a genuine connection. That is to say, please do not approach people "cold" and ask them out of the blue to connect with you, or for a favor like "help me get this job in your company" or, "please be my mentor". This may be perceived as disrespectful. People are busy, and they reserve their limited time for helping those they know and respect.

> Your best strategy is to start conversations with a genuine interest in the person and connect on a mutual interest. For example, you could review someone’s profile on LinkedIn and find something in common, or begin a conversation in-person with questions related to what the person does, how they got to where they are, and what challenges they face in their daily work. Make it about them, not you. Show that you are interested in them as a person, not that you want them to give you something. Actively listen to what they say and ask follow-up questions and even offer to help with something if you are in a position to do so. Think of the interaction like dating, you don’t ask someone to be your partner in the first question (do you? lol), so the same applies here. If your initial conversation goes well, then you can follow up with a request to connect.

Reviewer 3: ```suggestion
Your best strategy is to start conversations with a genuine interest in the person and connect on a mutual interest. For example, you could review someone’s profile on LinkedIn and find something in common, or begin a conversation in-person with questions related to what the person does, how they got to where they are, and what challenges they face in their daily work. Make it about them, not you. Show that you are interested in them as a person, not that you want them to give you something. Actively listen to what they say and ask follow-up questions and even offer to help with something if you are in a position to do so. Think of the interaction like dating: you don’t ask someone to be your romantic partner in your first conversation, right? The same applies here. If your initial conversation goes well, then you can follow up with a request to connect.
```
